script/blockchain/tezos.ts

```diff
@@ -1,14 +1,11 @@
-import axios from "axios";
 import * as eztz from "eztz-lib";
 import {
     validatorsList,
     getChainValidatorsPath,
-    getChainValidatorsListPath,
     getChainValidatorsAssets
 } from "../generic/repo-structure";
 import { Tezos } from "../generic/blockchains";
 import { readFileSync } from "../generic/filesystem";
-import { writeJsonFile } from "../generic/json";
 import { ActionInterface, CheckStepInterface } from "../generic/interface";
 import { ValidatorModel } from "../generic/validator-models";
 
```

Comment:
'BakingBadBaker' is defined but never used

Reply:
thanks, removed